src/main/resources/static/js/subscription.js

```diff
@@ -467,81 +458,74 @@ jQuery(document).ready(function () {
                     "render": function (data, type, row, meta) {
                         if (data == undefined || row == undefined) {
                             window.logMessages("Error: Subscription data is not defined");
-                            return ''
+                            return '';
                         }
-                        subscriptionOwner = row.ldapUserName
-                        subscriptionName = data.subscriptionName
+                        subscriptionOwner = row.ldapUserName;
```

Comment:
What if ldapUsername don't exist in Subscription?

Reply:
subscriptionOwner will be undefined, there is a check for that when determing if owner exist for subscription.